pytransform3d/test/test_batch_rotations.py

```diff
@@ -318,6 +347,28 @@ def test_axis_angles_from_matrices_output_variable():
     assert_array_almost_equal(A, A2_compact)
 
 
+def test_axis_angles_from_quaternions():
+    rng = np.random.default_rng(48322)
+    Q = pbr.norm_vectors(rng.standard_normal(size=(20, 4)))
```

Comment:
Similar case as in `norm_axis_angles` tests:

```suggestion
    num_instances = 20
    Q = pbr.norm_vectors(rng.standard_normal(size=(num_instances, 4)))
```